content/en/blog/2024/sig-arch-enhancements-spotlight.md

```diff
@@ -0,0 +1,152 @@
+---
+layout: blog
+title: "Spotlight on SIG Architecture: Enhancements"
+slug: sig-architecture-enhancements-2024
```

Comment:
The URL would be https://k8s.dev/blog/2024/09/18/sig-architecture-enhancements-2024/

```suggestion
slug: sig-architecture-enhancements
```

to avoid tautology.